definitions/family-private/json/AuthorisationCaseField.json

```diff
@@ -2581,5 +2581,208 @@
     "CaseFieldID": "OtherSafetyOrWelfareConcernsDetails",
     "UserRole": "caseworker-privatelaw-superuser",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "23/08/2023",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "ApplicantAttendedMIAMLabel",
```

Comment:
Labels do not actually need any authorisations. Discovered that from Jakub :)

Reply:
:o

Reply:
I have a feeling they do when you need to render a value, like:

```This is my ${name}```

because then user is in fact updating a label.

But maybe for a static text it's not needed.

Reply:
Oh okay yep makes sense